engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -76,4 +96,11 @@ public void resetCamera(ResetCameraEvent resetCameraEvent, EntityRef client, Aut
         }
         Location.attachChild(targetEntityForCamera, clientComponent.camera, cameraPosition, new Quat4f(Quat4f.IDENTITY));
```

Comment:
I thought it might be more clear that the camera position is the only thing that could be different when there is no existing location.  That is why there is only one call to attachChild.   Let me know if you think it would be more readable/understandable as two different calls to attach child.
I am not sure what case would be more likely "cameraLocation == null" or "cameraLocation != null",  specially because there is no guarantee that the camera entity will come with a LocationComponent.